src/sections/Meshery/Meshery-integrations/IntegrationsGrid.js

```diff
@@ -139,7 +150,8 @@ const IntegrationsGrid = ({ category, theme, count }) => {
               className={item.isSelected ? "items selected" : "items"}
               onClick={setFilter}
             >
-              {item.name}
+              <span>{item.name}</span>
+              <span> ({item.count}) </span>
```

Comment:
Don't use `span` here, this causes issues in filtering if the click is not exactly on the filter text, but in the gray area due to the comparison logic.

Just render it like
```
{`${item.name} (${item.count})`}
```

Reply:
I tried but it is making the items non-clickable

Reply:
You would need a `.split("(")[0]` as well 😅
As the value returned contains the count, so we need to segregate it out before comparing.

Reply:
Like how. Can you please show the code?

Reply:
if you're having problem with the syntax, you can refer to this https://www.w3schools.com/jsref/jsref_split.asp

Reply:
Not the syntax exactly but where we need to use the function in the code

Reply:
`selectedCategory = selectedCategory.split("(")[0];`

Reply:
Do I need to insert this after line 92 like this
```
const selectedCategory = event.target.innerHTML.includes("&amp;") ? event.target.innerHTML.replace("&amp;", "&") : event.target.innerHTML;
 selectedCategory = selectedCategory.split("(")[0];
```

Reply:
Yup, and change the type to `let` from `const`

Reply:
Actually, `trim` is also needed here since we are adding a ` `("space" in between the count and the category name)
i.e, `selectedCategory = selectedCategory.split("(")[0].trim()`;

Reply:
THe items are still not clickable

Reply:
okay it works now :)